### PR TITLE
VSTS14842 - New Loggly Standards

### PR DESCRIPTION
**Pull request by jonathanweyermann**
* Re-did new standard using jbuilder
* Followed, as best as possible from https://albertamotorassociation.atlassian.net/wiki/spaces/AC/pages/27951105/Loggly+Standards
* VSTS14842 - https://amaabca.visualstudio.com/blue_tech_debt/_workitems/edit/14842

* Fixed all the tests

### CODE REVIEW

**Comment by jessedoyle** [comment]
:book:

**Comment by jonathanweyermann** [comment]
So this looks like
image
There is a details column that only shows up in the raw message for now (loggly parses it as a string,   but we think it will eventually correct itself

**Comment by mvandenbeuken** [comment]
📖

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.5.1
```
Comment (jbabe) [inline .ruby-version]: I think it's good to have `2.5.1` support but should we set this to the highest version that our apps use? Also, should we add to the list of built rubies on Travis? https://github.com/amaabca/rest-client-jogger/blob/master/.travis.yml#L3

Comment (jbabe) [reply .ruby-version]: [list of current .ruby-versions](https://github.com/search?q=org%3Aamaabca+filename%3A.ruby-version&unscoped_q=filename%3A.ruby-version)

Comment (jessedoyle) [reply .ruby-version]: I definitely agree with adding it to the travis build matrix. Maybe we shouldn't have a `.ruby-version` file in this gem as it run across many different Ruby interpreter versions.

**Comment by jessedoyle** [comment]
@jbabe - I think I addressed your comment.

**Comment by jbabe** [comment]
> @jbabe - I think I addressed your comment.

Yep - looks good to me @jessedoyle

**Comment by jessedoyle** [comment]
:wave:

**Comment by mvandenbeuken** [comment]
🐘 🐘 🐘

**Pull request merged**